youtube_dl_gui/locale/zh_CN/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,582 @@
+msgid "SUDO password"
+msgstr "SUDO密码"
+
+#: optionsframe.py:418 optionsframe.py:819
+msgid "In order for the changes to take effect please restart {0}"
+msgstr "为使更改生效 请重新启动 {0}"
+
+#: optionsframe.py:419 optionsframe.py:820
+msgid "Restart"
+msgstr "重启"
+
+#: optionsframe.py:466
+msgid "high"
+msgstr "高"
+
+#: optionsframe.py:466
+msgid "low"
+msgstr "低"
+
+#: optionsframe.py:466
+msgid "mid"
+msgstr "中"
+
+#: optionsframe.py:471
+msgid "Video formats"
+msgstr "视频格式"
+
+#: optionsframe.py:474
+msgid "Audio formats"
+msgstr "音频格式"
+
+#: optionsframe.py:477
+msgid "Post-Process options"
+msgstr "后处理选项"
+
+#: optionsframe.py:478
+msgid "Keep original files"
+msgstr "保留原始文件"
+
+#: optionsframe.py:479
+msgid "Extract audio from video file"
+msgstr "从视频中解压出音频"
+
+#: optionsframe.py:480
+msgid "Embed thumbnail in audio file"
+msgstr "在音频中封入封面"
+
+#: optionsframe.py:481
+msgid "Add metadata to file"
+msgstr "将metadata写入文件"
+
+#: optionsframe.py:483
+msgid "Audio quality"
+msgstr "音频质量"
+
+#: optionsframe.py:541
+msgid "English"
+msgstr "英语"
+
+#: optionsframe.py:542
+msgid "French"
+msgstr "法语"
+
+#: optionsframe.py:543
+msgid "German"
+msgstr "德语"
+
+#: optionsframe.py:544
+msgid "Greek"
+msgstr "希腊语"
+
+#: optionsframe.py:545
+msgid "Hebrew"
+msgstr "希伯来语"
+
+#: optionsframe.py:546
+msgid "Italian"
+msgstr "意大利语"
+
+#: optionsframe.py:547
+msgid "Portuguese"
+msgstr "葡萄牙语"
+
+#: optionsframe.py:548
+msgid "Russian"
+msgstr "俄语"
+
+#: optionsframe.py:549
+msgid "Spanish"
+msgstr "西班牙语"
+
+#: optionsframe.py:550
+msgid "Swedish"
+msgstr "瑞典语"
+
+#: optionsframe.py:551
+msgid "Turkish"
+msgstr "土耳其语"
+
+#: optionsframe.py:567
+msgid "None"
+msgstr "无"
+
+#: optionsframe.py:568
+msgid "Automatic subtitles (YOUTUBE ONLY)"
+msgstr "自动生成字幕(仅限YOUTUBE)"
+
+#: optionsframe.py:569
+msgid "All available subtitles"
+msgstr "所有可用字幕"
+
+#: optionsframe.py:570
+msgid "Subtitles by language"
+msgstr "根据语言选择字幕"
+
+#: optionsframe.py:576
+msgid "Subtitles"
+msgstr "字幕"
+
+#: optionsframe.py:580
+msgid "Subtitles options"
+msgstr "字幕选项"
+
+#: optionsframe.py:581
+msgid "Embed subtitles into video file (mp4 ONLY)"
+msgstr "将字幕封入视频(仅限mp4视频)"
+
+#: optionsframe.py:583
+msgid "Playlist"
+msgstr "播放列表"
+
+#: optionsframe.py:589 optionsframe.py:594
+msgid "Max"
+msgstr "最大"
+
+#: optionsframe.py:592
+msgid "Filesize"
+msgstr "文件大小"
+
+#: optionsframe.py:597
+msgid "Min"
+msgstr "最小"
+
+#: optionsframe.py:726
+msgid "Retries"
+msgstr "重试次数"
+
+#: optionsframe.py:729
+msgid "Authentication"
+msgstr "授权"
+
+#: optionsframe.py:731
+msgid "Username"
+msgstr "用户名"
+
+#: optionsframe.py:733
+msgid "Password"
+msgstr "密码"
+
+#: optionsframe.py:735
+msgid "Video password"
+msgstr "视频密码"
+
+#: optionsframe.py:738
+msgid "Network"
+msgstr "网络"
+
+#: optionsframe.py:740
+msgid "Proxy"
+msgstr "代理"
+
+#: optionsframe.py:742
+msgid "User agent"
+msgstr "用户代理"
+
+#: optionsframe.py:744
+msgid "Referer"
+msgstr "引用"
+
+#: optionsframe.py:747
+msgid "Logging"
+msgstr "登录中"
```

Comment:
"Logging" means *keeping track of some event* or *recording* that event. Google translates your *msgstr* string to "Logging in" which does not match the meaning of the *msgid* string. Are you sure this is correct?